corehq/apps/userreports/reports/data_source.py

```diff
@@ -126,16 +127,17 @@ def column_warnings(self):
     def get_data(self):
         try:
             ret = super(ConfigurableReportDataSource, self).get_data()
-            for report_column in self.column_configs:
-                report_column.format_data(ret)
         except (
             ColumnNotFoundException,
```

Comment:
Seems like at the very least `ColumnWithNameNotFoundException` could subclass `ColumnNotFoundException`, or maybe they both have a common ancestor which would make sense to use here.